src/managers/guildChannels.ts

```diff
@@ -116,4 +116,8 @@ export class GuildChannelsManager extends BaseChildManager<
     )
     return this
   }
+
+  async keys(): Promise<string[]> {
+    return (await this.array()).map((c) => c.snowflake.id)
```

Comment:
This has a performance hit where all channel payloads are first converted into structures, and then you're using snowflake getter which is redundant as well, since there is directly `id` on channel.
You should use `client.cache.array` to get a list of raw payloads and filter based on that.